modules/deploy/pages/deployment-option/cloud/aws-privatelink.adoc

```diff
@@ -0,0 +1,394 @@
+----
+
+== Enable PrivateLink endpoint service for existing clusters
+
+. In the Redpanda Cloud UI, go to the cluster overview and copy the cluster ID from the **Details** section.
++
+[,bash]
+----
+CLUSTER_ID=<cluster_id>
+----
+
+. Make a `PATCH /v1beta1/clusters/{cluster.id}` request to update the cluster with the Redpanda Private Link Endpoint Service enabled.
++
+In the example below, make sure to set your own value for the following field:
++
+--
+- `allowed_principals`: Amazon Resource Names (ARNs) for the AWS principals allowed to access the endpoint service. For example, for all principals in an account, use `arn:aws:iam::account_id:root`. See https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html#add-remove-permission[Configure an endpoint service] for details.
+--
++
+[,bash]
+----
+CLUSTER_PATCH_BODY=`cat << EOF
+{
+  "private_link": {
+     "enabled": true,
+     "aws": {
+        "allowed_principals": ["<principal_1>","<principal_2>", ...]
+     }
+  }
+}
+EOF`
+
+curl -vv -X PATCH \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer $AUTH_TOKEN" \
+  -d "$CLUSTER_PATCH_BODY" $PUBLIC_API_ENDPOINT/v1beta1/clusters/$CLUSTER_ID
+----
+
+. Before proceeding, check the state of the Update Cluster operation by calling https://docs.api.redpanda.com/#get-/v1beta1/operations/-id-[GET /v1beta1/operations/{id}], passing the operation ID returned from Update Cluster call. When the state is `STATE_READY`, proceed to the next step.
+
+. Check the service state by calling https://docs.api.redpanda.com/#get-/v1beta1/clusters/-id-[`GET /v1beta1/clusters/{id}`]. The `service_state` in the `private_link.status` response object needs to be `Available` for you to <<connect-to-privatelink-endpoint-service,connect to the service>>.
++
+[,bash]
+----
+curl -X GET \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer $AUTH_TOKEN" \
+    $PUBLIC_API_ENDPOINT/v1beta1/clusters/$CLUSTER_ID | jq '.private_link.status.aws | {service_name, service_state}'
+----
+
+== Connect to PrivateLink endpoint service
+
+=== Get cluster domain
+
+Get the domain (`cluster_domain`) of the cluster from the cluster details in the Redpanda Cloud UI.
+
+For example, if the Bootstrap server URL is: `seed-3da65a4a.cki01qgth38kk81ard3g.fmc.dev.cloud.redpanda.com:9092`, then `cluster_domain` is: `cki01qgth38kk81ard3g.fmc.dev.cloud.redpanda.com`.
+
+[,bash]
+----
+CLUSTER_DOMAIN=<cluster_domain>
+----
+
+=== Get name of PrivateLink endpoint service
+
+You need the service name to <<create-vpc-endpoint,create VPC private endpoints>>. Run the following command to get the service name:
+
+[,bash]
+----
+PL_SERVICE_NAME=`curl -X GET \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer $AUTH_TOKEN" \
+  $PUBLIC_API_ENDPOINT/v1beta1/clusters/$CLUSTER_ID | jq -r .private_link.status.aws.service_name`
+----
+
+=== Create client VPC
+
+If you are not using an existing VPC, you must create a new one. 
```

Comment:
There is a gotcha we should explicitly document somewhere. If the VPC where clients live already has a VPC Peering against the cluster's VPC, enabling Private Links will make those clients fail to connect until they are updated to use the Private Link endpoints.    They will no longer be able to use the peering to connect.

Reply:
@c4milo Could we add a banner/admonition right in this spot? Please let me know if the phrasing is correct:

```
CAUTION: While you can use your existing client VPC, if it already has a peering connection against the cluster's VPC, clients will no longer be able to use peering to connect. You must update those clients to use the PrivateLink endpoints.
```